toolbox/util/githubUtils.go

```diff
@@ -87,3 +88,47 @@ func GetAPITokenFromFile(tokenFile string) (string, error) {
 	token := strings.TrimSpace(string(b[:]))
 	return token, nil
 }
+
+// CloneRepoCheckoutBranch removes previous repo, clone to local machine,
+// change directory into the repo, and checkout the given branch
+func CloneRepoCheckoutBranch(gclient *GithubClient, repo, branch string) error {
+	if err := os.RemoveAll(repo); err != nil {
+		return err
+	}
+	if _, err := ShellSilent(
+		"git clone " + gclient.Remote(repo)); err != nil {
+		return err
+	}
+	if err := os.Chdir(repo); err != nil {
+		return err
+	}
+	_, err := Shell("git checkout " + branch)
+	return err
+}
+
+// RemoveLocalRepo deletes the local git repo just cloned
+func RemoveLocalRepo(repo string) error {
+	if err := os.Chdir(".."); err != nil {
```

Comment:
I am kind of uncomfortable with those chdir, as if I call those method in a random order I might not know where I end up. Could you see how difficult it would be to use the github api to changes files ? One option is that for some files changes (ie increasing version) we might not need to create a PR, we can just have the bot do it direclty.